tdrs-backend/Pipfile

```diff
@@ -45,6 +45,10 @@ pyjwt = "==2.4.0"
 requests = "==2.27.1"
 wait-for-it = "==2.2.0"
 requests-mock = "==1.9.3"
+elasticsearch = "==7.13.4"
+django-elasticsearch-dsl = "==7.1.1"
+django-elasticsearch-dsl-drf = "==0.22.5"
+requests-aws4auth = "==1.1.2"
 
```

Comment:
new library additions for the elastic changes. *NOTE* - `elasticsearch` MUST stay at version `7.13.4` as it is the last version of the library compatible with AWS elastic search, which we use through Cloud.Gov

this article has some good information on the breaking changes
https://towardsaws.com/elasticsearch-the-server-is-not-a-supported-distribution-of-elasticsearch-252abc1bd92

`django-elasticsearch-dsl` has likewise been downgraded to the last version to support `elasticsearch v7.13.4`